kubernetes/watch/watch_util.h

```diff
@@ -0,0 +1,13 @@
+#ifndef _WATCH_UTIL_H
+#define _WATCH_UTIL_H
+
+#ifdef  __cplusplus
+extern "C" {
+#endif
+
+void kubernets_watch_handler(void** pData, long* pDataLen, void (*event_hander)(const char*));
```

Comment:
Do we want to `typedef` the event handler callback?

Reply:
Sure. It's better. I will do it.